pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -683,7 +683,7 @@
     "safe_mode" = ps: with ps; [ aiohttp-cors hass-nabucasa]; # missing inputs: home-assistant-frontend
     "saj" = ps: with ps; [ ]; # missing inputs: pysaj
     "salt" = ps: with ps; [ ]; # missing inputs: saltbox
-    "samsungtv" = ps: with ps; [ ]; # missing inputs: samsungctl[websocket] samsungtvws[websocket]
+    "samsungtv" = ps: with ps; [ samsungctl samsungtvws];
```

Comment:
nit
```suggestion
    "samsungtv" = ps: with ps; [ samsungctl samsungtvws ];
```

Reply:
Yeah, that's something we need to fix at some point in the `parse_requirements.py` script. That diff is autogenerated.

Reply:
Fixed.